parm/config/config.vrfy

```diff
@@ -101,7 +104,7 @@ fi
 
 if [ $VRFYRAD = "YES" -o $VRFYMINMON = "YES" -o $VRFYOZN = "YES" ]; then
 
-    export envir="para"
+    export envir="prod"
```

Comment:
Why the change?

Reply:
The radmon ,minmon, and oznmom used internal variable envir that need to be set on this level.

Reply:
That's why it is set here, but why was it _changed_

Reply:
I will drop this change. Do you have any other question on this PR before I update it?

Reply:
Just the general comment about the j-jobs, but I didn't really go through those carefully, though I assume they are just what was previously in the rocoto/<task> script.

Reply:
Dropped